src/templates/default/values.template.yml

```
@@ -14,3 +14,4 @@ containerResources:
     memory: 256M
     cpu: 100m
 envSecretName: {{appName}}__STAGE__-secret-env
+sentry: {{useSentry}}
```

```suggestion
sentry: {{sentry}}
```
(Aangezien het al een boolean is)